opencti-platform/opencti-front/src/private/components/common/stix_core_relationships/StixCoreRelationshipCreationFromEntity.jsx

```diff
@@ -569,26 +569,26 @@ const StixCoreRelationshipCreationFromEntity = (props) => {
       toEntities = [sourceEntity];
     }
     return (
-      <UserContext.Consumer>
-        {({ schema }) => {
-          const relationshipTypes = R.filter(
-            (n) => R.isNil(allowedRelationshipTypes)
-              || allowedRelationshipTypes.length === 0
-              || allowedRelationshipTypes.includes('stix-core-relationship')
-              || allowedRelationshipTypes.includes(n),
-            resolveRelationsTypes(
-              fromEntities[0].entity_type,
-              toEntities[0].entity_type,
-              schema.schemaRelationsTypesMapping,
-            ),
-          );
-          return (
-            <>
-              <div className={classes.header}>
-                <IconButton
-                  aria-label="Close"
-                  className={classes.closeButton}
-                  onClick={() => handleClose()}
+        <UserContext.Consumer>
+          {({ schema }) => {
+            const relationshipTypes = R.filter(
+              (n) => R.isNil(allowedRelationshipTypes)
+                    || allowedRelationshipTypes.length === 0
+                    || allowedRelationshipTypes.includes('stix-core-relationship')
+                    || allowedRelationshipTypes.includes(n),
+              resolveRelationsTypes(
+                fromEntities[0].entity_type,
+                toEntities[0].entity_type,
+                schema.schemaRelationsTypesMapping,
+              ),
+            );
+            return (
+                <>
+                  <div className={classes.header}>
+                    <IconButton
+                        aria-label="Close"
+                        className={classes.closeButton}
+                        onClick={() => handleClose()}
```

Comment:
I don't see anything that changed in this file, except identation (and not the right one)